include/aunit/reporters/aunit-reporter-xml.adb

```diff
@@ -93,16 +94,16 @@ package body AUnit.Reporter.XML is
 
       Put_Line (File, "  <Statistics>");
       Put      (File, "    <Tests>");
-      Put (File, Integer (Test_Count (R)), 0);
+      Put (File, Trim (Test_Count (R)'Img, Left));
       Put_Line (File, "</Tests>");
       Put      (File, "    <FailuresTotal>");
-      Put (File, Integer (Failure_Count (R)) + Integer (Error_Count (R)), 0);
+      Put (File, Trim (Count_Type'Image (Failure_Count (R) + Error_Count (R)), Left));
```

Comment:
This fails to build:

aunit-reporter-xml.adb:100:24: "Count_Type" is not visible
aunit-reporter-xml.adb:100:24: non-visible declaration at ada_containers.ads:20